### DISCUSSION TITLE
Using a Nextion display or NSPanel

### DISCUSSION THREAD

**Discussion by selvasalus** [signalkraft/mypyllant-component#30] [General]
Hello, congratulations on the integration, it is great.
How can I change the temperature setpoint from a Nextion screen?
I can capture the rest of the values without problem and change the mode, but I cannot increase the setpoint.
Can be done?
Thank you very much and greetings.

**Comment by signalkraft** [comment]
You should be able to use the default climate controls in home assistant. What happens when you change the target temperature depends on your heating mode. In time controlled mode, it sets a quick veto. In manual mode, it sets the manual setpoint (permanently).

Seeing this change in HA can take a little bit.
